### ISSUE TITLE
Issues with OSX Mojave

### ISSUE THREAD

**Issue by btnewton** [aseprite/aseprite#1960]
Aseprite v1.2.10 Beta & v1.2.9 on Mac Mojave 10.14.2

I've found a couple of issues running Aseprite on Mojave.

- holding alt to use the eyedropper doesn't do anything. All shortcuts I've tried that don't require holding a button down seem to work though.
- the rectangular marque, lasso and magic wand tools (but not the other marque tools) duplicate the selected pixels rather than moving them.

I've tried reinstalling both versions (from Humble Bundle). Neither of these problems existed before I updated to Mojave.

Please let me know if you need any other information!

**Comment by dacap** [comment]
I have Mojave 10.14.1 (still not updated to 10.14.2) and the Alt key works, I'll give a try to 10.14.2 next week, just in case. Try to reset all your keyboard shortcuts from Edit > Keyboard Shortcuts > Reset (you can Edit > Keyboard Shortcuts > Export them before just in case and send it to me to support@aseprite.org ).

**Comment by btnewton** [comment]
Thanks for the quick response! Reseting the keyboard shortcuts did not change the behavior. I emailed my exported shortcuts to support. When holding the alt key the eye dropper flashes on screen for ~1 frame before disappearing and returning to the previous tool.

**Comment by dacap** [comment]
I've updated to 10.14.2 but the Alt key is working correctly. Do you have some special driver/device? (magic mouse, wacom stylus, etc.)

**Comment by btnewton** [comment]
Good suggestion! I recently purchased a Wacom tablet (the Wacom Intuos) and **I now believe the Wacom software was the issue**. I mapped two of the tablet buttons to "m" and alt. This caused issues _even when the tablet was unplugged_ which was misleading. Disabling all tablet buttons in System Preferences > Wacom Tablet > Options immediately solved the issue in Aseprite - I didn't even have to restart Aseprite. Curiously, after I re-enabled the buttons the issue did not return. 

This solves the issue for me so you're welcome to close it. However, I'm happy to provide more information or try some things out if it's helpful.
